lib/new_sets.bzl

```diff
@@ -217,6 +218,19 @@ def _repr(s):
     """
     return repr(s._values.keys())
 
+_SET_TYPE = type(_make())
```

Comment:
This should be in types.bzl, alongside the other is_? predicates.

Reply:
For some reason GitHub is stating a change is requested

Reply:
Done